test/test_ui_commands.py

```diff
@@ -84,6 +84,25 @@ def test_query_album(self):
         self.check_do_query(0, 2, album=True, also_items=False)
 
 
+class FieldsTest(_common.TestCase):
+    def setUp(self):
+        super(FieldsTest, self).setUp()
+
+        self.io.install()
+
+        self.libdir = os.path.join(self.temp_dir, 'testlibdir')
+        os.mkdir(self.libdir)
+
+        # Add a file to the library but don't copy it in yet.
+        self.lib = library.Library(':memory:', self.libdir)
```

Comment:
If you inherit from `_common.LibTestCase` instead of plain `_common.TestCase`, you'll get `self.lib` for free.